pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/pdf/DownloadPDF.java

```diff
@@ -84,39 +86,60 @@ public DownloadPDF(@NonNull String requestBody) {
 
     public Optional<byte[]> getPDFs(long userIdRequest, String realm, String requestBody) {
         UserDto user = new UserDao().get(userIdRequest).orElseThrow();
-        Optional<byte[]> pdfBytes = Optional.empty();
+        Optional<byte[]> maybePdf = Optional.empty();
         if (StringUtils.isNotBlank(this.ddpParticipantId)) {
             DDPInstance ddpInstance = DDPInstance.getDDPInstance(realm);
             if (ddpInstance != null && StringUtils.isNotBlank(ddpParticipantId)) {
-                String fileName = "";
```

Comment:
I found a bug that this removed the name of the file type when saving file to the bucket (already on dev) so I changed it use configName from now